2024-01-09 17:49:33,611 INFO    MainThread:8336 [wandb_setup.py:_flush():76] Current SDK version is 0.15.2
2024-01-09 17:49:33,612 INFO    MainThread:8336 [wandb_setup.py:_flush():76] Configure stats pid to 8336
2024-01-09 17:49:33,612 INFO    MainThread:8336 [wandb_setup.py:_flush():76] Loading settings from /Users/dw/.config/wandb/settings
2024-01-09 17:49:33,612 INFO    MainThread:8336 [wandb_setup.py:_flush():76] Loading settings from /Users/dw/code/pytorch/gammassl_public/training/wandb/settings
2024-01-09 17:49:33,612 INFO    MainThread:8336 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-09 17:49:33,612 INFO    MainThread:8336 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-09 17:49:33,612 INFO    MainThread:8336 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'training/train.py', 'program': '/Users/dw/code/pytorch/gammassl_public/training/train.py'}
2024-01-09 17:49:33,612 INFO    MainThread:8336 [wandb_init.py:_log_setup():507] Logging user logs to /Users/dw/code/pytorch/gammassl_public/training/wandb/run-20240109_174933-f25m70xy/logs/debug.log
2024-01-09 17:49:33,612 INFO    MainThread:8336 [wandb_init.py:_log_setup():508] Logging internal logs to /Users/dw/code/pytorch/gammassl_public/training/wandb/run-20240109_174933-f25m70xy/logs/debug-internal.log
2024-01-09 17:49:33,613 INFO    MainThread:8336 [wandb_init.py:init():547] calling init triggers
2024-01-09 17:49:33,613 INFO    MainThread:8336 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {'gamma_reject_prop': 0.5, 'data_aug': 'candr', 'no_colour': False, 'use_sax_png_dataset': False, 'use_dinov1': False, 'frozen_target_save_path': None, 'frozen_target': False, 'proj_query_pup': False, 'enc_td_no_query_grad': False, 'td_no_query_grad': False, 'sharpen_temp': None, 'ema_update_every': 1, 'ema_beta': 0.999, 'use_ema_target_net': False, 'target_temp_factor': 0.1, 'use_symmetric_branches': False, 'validate_only': False, 'qual_validate': False, 'qual_output_dir': '/Users/dw/processed_results/qual_results', 'skip_uniformity': False, 'skip_projection': False, 'm2f_query_test': False, 'nheads': 4, 'num_points': 12544, 'run_sup_task': False, 'random_mask_prob': None, 'run_masking_task': False, 'use_scannet_twice': False, 'proto_seg_for_sup': False, 'imagenet_dataroot': '', 'scannet_dataroot': None, 'scannet': False, 'weird_dataroot': None, 'no_unlabelled': False, 'sunrgbd_dataroot': None, 'sunrgbd': False, 'val_all_sax': False, 'vit_size': 'small', 'n_train_segs': 4, 'n_val_segs': 4, 'use_wandb': True, 'train_vit': False, 'exp_name': None, 'lora_rank': None, 'dino_repo_path': '/Users/dw/code/pytorch/dinov2', 'dino_path': '/Users/dw/code/pytorch/gammassl/models/dinov2.pth', 'dinov1_repo_path': None, 'dinov1_path': None, 'cityscapes_only': False, 'encoder': 'dino_repo', 'use_vit_adapter': False, 'decode_head': 'setr', 'use_dino': False, 'use_fake_data': False, 'use_fixed_random_init': False, 'model_config_path': '', 'model_weight_decay': 0, 'no_filtering': False, 'use_resize_noise': False, 'uniformity_kernel_size': 4, 'uniformity_stride': 4, 'include_void': True, 'max_uncertainty': 1, 'threshold_type': 'linear', 'num_thresholds': 500, 'lr': 0.0005, 'lr_backbone': None, 'warmup_ratio': None, 'n_warmup_iters': None, 'lr_policy': None, 'lr_mult': 1, 'decay_mult': 1, 'total_iters': 160000, 'w_c': 1, 'w_s': 1, 'w_gamma': 1, 'w_u': 1, 'w_p': 1, 'w_m': 1, 'w_mask': 1, 'w_dice': 1, 'w_ce': 1, 'batch_size': 24, 'n_epochs': 1000, 'use_class_weights_ssl': False, 'use_class_weights_sup': False, 'no_transforms': False, 'overfit_on_batch': False, 'feature_len': None, 'prototype_len': 256, 'intermediate_dim': 256, 'feature_df': 4, 'cityscapes_dataroot': '/Users/dw/data/', 'unlabelled_dataroot': '/Users/dw/data/bdd100k', 'wilddash_dataroot': '/Users/dw/data/wilddash', 'sax_raw_dataroot': '/Users/dw/data/sax_raw', 'sax_labelled_dataroot': '/Users/dw/data/sax_labelled', 'sensor_models_path': '/Users/dw/code/lut/sensor-models', 'cityscapes_class_weights_path': '/Volumes/scratchdata/dw/cityscapes_class_weights_19c.pkl', 'benchmarks_results_dir': '/Volumes/scratchdata/dw/project_sax_results_tau_eq_1/results', 'sax_domain': 'london', 'min_crop_ratio': 2, 'max_crop_ratio': 3, 'big_crop_size': 208, 'device': 'cpu', 'use_cpu': True, 'gpu_no': '0', 'num_workers': 0, 'use_deep_features': True, 'seg_net_config_path': None, 'use_imagenet_norm': True, 'imagenet_save_path': None, 'temperature': 0.07, 'log_dir': None, 'log_every': 1, 'visdom_port': 8097, 'output2visdom': False, 'full_validation_every': 1000, 'skip_validation': True, 'save_every': 1, 'network_destination': None, 'save_path': None, 'prototypes_path': None}
2024-01-09 17:49:33,613 INFO    MainThread:8336 [wandb_init.py:init():596] starting backend
2024-01-09 17:49:33,613 INFO    MainThread:8336 [wandb_init.py:init():600] setting up manager
2024-01-09 17:49:33,618 INFO    MainThread:8336 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-01-09 17:49:33,625 INFO    MainThread:8336 [wandb_init.py:init():606] backend started and connected
2024-01-09 17:49:33,647 INFO    MainThread:8336 [wandb_init.py:init():700] updated telemetry
2024-01-09 17:49:33,739 INFO    MainThread:8336 [wandb_init.py:init():737] communicating run to backend with 60.0 second timeout
2024-01-09 17:49:34,200 INFO    MainThread:8336 [wandb_run.py:_on_init():2177] communicating current version
2024-01-09 17:49:34,300 INFO    MainThread:8336 [wandb_run.py:_on_init():2186] got version response upgrade_message: "wandb version 0.16.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-01-09 17:49:34,300 INFO    MainThread:8336 [wandb_init.py:init():787] starting run threads in backend
2024-01-09 17:49:41,417 WARNING MainThread:8336 [wandb_init.py:init():1172] interrupted
Traceback (most recent call last):
  File "/Users/dw/miniconda3/envs/dinov2/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 1150, in init
    run = wi.init()
  File "/Users/dw/miniconda3/envs/dinov2/lib/python3.9/site-packages/wandb/sdk/wandb_init.py", line 799, in init
    run_start_result = run_start_handle.wait(timeout=30)
  File "/Users/dw/miniconda3/envs/dinov2/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/Users/dw/miniconda3/envs/dinov2/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/Users/dw/miniconda3/envs/dinov2/lib/python3.9/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/Users/dw/miniconda3/envs/dinov2/lib/python3.9/threading.py", line 581, in wait
    signaled = self._cond.wait(timeout)
  File "/Users/dw/miniconda3/envs/dinov2/lib/python3.9/threading.py", line 316, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-01-09 17:49:44,749 WARNING MsgRouterThr:8336 [router.py:message_loop():77] message_loop has been closed
